app/coffee/RedisManager.coffee

```diff
@@ -168,6 +169,8 @@ module.exports = RedisManager =
 						ops = jsonOps.map (jsonOp) -> JSON.parse jsonOp
 					catch e
 						return callback(e)
+					timeSpan = timer.done()
+					return callback(new Error("redis getPreviousDocOps exceeded timeout")) if timeSpan > MAX_REDIS_REQUEST_LENGTH
```

Comment:
Quite a long line, and it looks on first glance like we're always returning an error at the end of the method (due to trailing if). Can we split into:

```
if ...
   return ...
```